core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/columns/BaseColumn.kt

```diff
@@ -18,7 +19,7 @@ import kotlin.reflect.KType
  *
  * @param T type of values contained in column.
  */
-public interface BaseColumn<out T> : ColumnReference<T> {
+public interface BaseColumn<out T> : ColumnReference<T>, GenericColumn {
```

Comment:
@koperagen What's `GenericColumn` doing here btw? 
It's fine to have the interface as its supertype to just get the name of a column, but it lives in ./impl/api/insert.kt which breaks our structure. Probably best to move `GenericColumn(Group)` to ./columns

Reply:
Ah I see it's mentioned in the comment! thanks